linux.mk

```diff
@@ -214,21 +196,17 @@ export MTU ?= 1500
 export MSS ?= 1500
 export PEER ?= server
 export TEST ?= udp-push-pop
-export TEST_INTEGRATION ?= tcp-test
+export TEST_INTEGRATION ?= tcp-tests
```

Comment:
Should this also include udp-tests?

Reply:
This is the default value for the integration test. This line changed becuase I changed `tcp-test` to `tcp-tests` (plural). The CI pipeline does run `udp-tests`.